wordpress-mobile/gutenberg-mobile
# Fix crash on quotes when writing on the last line.

This PR fix the issue when a some quote is written on the last line of the quote block on iOS.


To test:
 - Start the demo app
 - Insert a quote block
 - Select the quote part and move to the last line
 - Write content there
 - Switch to HTML mode
 - Check that no div is present and validation works fine.
 - Switch to visual
 - Check that all works ok

Update release notes:
